documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,183 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+:toc:
+
+In the context of application development, Maven provides a standardized build system for projects. One of the main benefits of using Maven with your project is that it facilitates fetching dependencies from one or more repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
```

Comment:
I agree the general concepts (modules, dep. mgmt) are not specific to Eclipse based IDE, but all the instructions hereafter are really about using Eclipse wizards and features